lib/vast/Conversion/FromHL/ToHLBI.cpp

```diff
@@ -0,0 +1,434 @@
+// Copyright (c) 2024-present, Trail of Bits, Inc.
+
+#include "vast/Conversion/Passes.hpp"
+
+#include "vast/Util/Warnings.hpp"
+
+VAST_RELAX_WARNINGS
+#include <clang/Basic/Builtins.h>
+#include <mlir/IR/PatternMatch.h>
+#include <mlir/Transforms/DialectConversion.h>
+#include <mlir/Transforms/GreedyPatternRewriteDriver.h>
+VAST_UNRELAX_WARNINGS
+
+#include "PassesDetails.hpp"
+#include "vast/Conversion/Common/Block.hpp"
+#include "vast/Conversion/Common/Passes.hpp"
+#include "vast/Conversion/Common/Patterns.hpp"
+#include "vast/Conversion/Common/Rewriter.hpp"
+
+#include "vast/Util/Common.hpp"
+#include "vast/Util/DialectConversion.hpp"
+#include "vast/Util/TypeList.hpp"
+
+#include "vast/Dialect/Builtin/Dialect.hpp"
+#include "vast/Dialect/Builtin/Ops.hpp"
+#include "vast/Dialect/HighLevel/HighLevelDialect.hpp"
+
+#include "../PassesDetails.hpp"
+
+#include <memory>
+
+namespace vast::conv {
+
+    struct true_visitor
```

Comment:
make its methods constexpr noexcept

Reply:
I changed it o the mlir::LogicalResult which isn't literal type so now it can not be constexpr